release-notes/opensearch-dashboards-reports.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,40 @@
+### **Version 1.2.0.0 Release Notes**
```

Comment:
could you remove the `**`? I don't think we use this

Reply:
Yeah i'll remove it